package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "grace-shopper-team-c",
+  "version": "1.0.0",
+  "description": "",
+  "main": "db.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "test:dev": "DATABASE_URL=postgres://localhost/grace_shopper_db mocha --watch",
+    "postinstall": "npm run webpack",
+    "webpack": "webpack",
+    "webpack:dev": "webpack --watch --mode=development",
+    "start:dev": "npm run webpack:dev & SECRET=FOO nodemon server.js --ignore dist/ --ignore src/",
```

Comment:
We should not assume other developers have `nodemon`. We should install `nodemon` as a dev dependency, and this script should use the installed dev dependency by using the binaries in `node_modules`

e.g. `./node_modules/.bin/nodemon server.js`